src/renderer.ts

```diff
@@ -83,7 +83,11 @@ getIpcApi()
 		window.appGlobals = appGlobals;
 
 		// Show warning if running an ARM64 translator
-		if ( appGlobals.arm64Translation && ! localStorage.getItem( 'dontShowARM64Warning' ) ) {
+		if (
+			appGlobals.platform === 'darwin' &&
```

Comment:
I considered limiting it to `appGlobals.platform !== 'win32'`, but I decided to exclude Linux as well.